src/resources/views/crud/buttons/delete.blade.php

```diff
@@ -47,9 +47,9 @@ className: "bg-danger",
 			          	// Show an error alert
 			              swal({
 			              	title: "{!! trans('backpack::crud.delete_confirmation_not_title') !!}",
-			              	text: "{!! trans('backpack::crud.delete_confirmation_not_message') !!}",
+                            text: result['error'] ? result['error'] : "{!! trans('backpack::crud.delete_confirmation_not_message') !!}",
```

Comment:
I think these are meant to be:
```php
text:  "{!! result['error'] ? result['error'] : trans('backpack::crud.delete_confirmation_not_message') !!}",
```

not
```php
text: result['error'] ? result['error'] : "{!! trans('backpack::crud.delete_confirmation_not_message') !!}",
```

Right? I can edit inline don't worry, just confirm.

Reply:
Also, if we do this. Wouldn't it trigger a PHP error if ```result['error']``` is not set? Which would be in most cases - when the error message is not customized.

Perhaps this would avoid the PHP error message
```php
text:  "{!! result['error'] ?? trans('backpack::crud.delete_confirmation_not_message') !!}",
```

Reply:
Hi my implementation is correct, because the result is returned by ajax in javascript, so it is not a blade property.